data_steward/cdr_cleaner/cleaning_rules/remove_invalid_procedure_source_records.py

```diff
@@ -38,12 +43,20 @@
  SELECT
```

Comment:
Lines 41-43 concern me.  The comment says "not in" but the condition being applied is `IN`.  Is this correct?  FYI:  I really don't know.

Reply:
this is correct since it's creating the sandbox table. It is writing all the invalid rows to the corresponding sandbox.